package.json

```diff
@@ -34,6 +34,8 @@
     "find-circular-dependencies": "madge -b src/ --webpack-config webpack.config.js -c --extensions js,jsx src/main.jsx",
     "posttest": "yarn find-circular-dependencies",
     "start": "npm run start:browser",
+    "watch": "npm run watch:browser",
+    "watch:browser": "NODE_ENV=browser:development npm run commons:watch",
```

Comment:
can you explain why the need of watch instead of start? Also maybe it should have a dedicated commit.

Reply:
j'ai supprimé, il y avait des soucis avec le start qui fait que le hotreload ne marche pas toujours, donc besoin de refresh quand même, mais avec le watch c'est pire, donc inutile